test/index.js

```diff
@@ -0,0 +1,141 @@
+/* global describe, it */
+import assert from 'assert';
+import SimpleTestServer from '../';
+import http from 'http';
+
+describe('SimpleTestServer', function () {
+	let server;
+	it('should should load the SimpleTestServer class', function () {
+		assert.ok(SimpleTestServer, 'SimpleTestServer class loaded');
+		server = new SimpleTestServer();
+		assert.ok(server, 'server initiated');
+	});
+	it('should should start the server', function (done) {
+		server.start(done);
```

Comment:
You should never rely on another test completing to setup the state for this test.  It makes for brittle tests because one can fail and cause a cascade.  If you need a server in each test just use `beforeEach`:

```
var server;
beforeEach(function () {
  server = new SimpleTestServer();
});
afterEach(function() {
  server = null;
});
```

Nulling out the server means you dont bleed any state over accidentally.